dataland-frontend/src/components/forms/parts/fields/NumberFormField.vue

```diff
@@ -2,10 +2,10 @@
   <div class="form-field">
     <UploadFormHeader :name="displayName" :explanation="info" :is-required="required" />
     <FormKit
-      type="number"
+      type="text"
```

Comment:
Just out of curiosity:
Why have you done this?  Can't FormKit do validations on number-type form fields?

Reply:
Not always. Let's talk about this.